Jar/src/velma/Storage.java

```diff
@@ -0,0 +1,115 @@
+package velma;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Scanner;
+import velma.task.Task;
```

Comment:
I like that you have used lowercase for package names as per the coding standard